exchanges/orderbook/unsafe.go

```diff
@@ -62,3 +66,100 @@ func (d *Depth) GetUnsafe() Unsafe {
 		LastUpdated:    &d.options.lastUpdated,
 	}
 }
+
```

Comment:
Do any of the below functions need to utilise locks? I recognise that this is `Unsafe` and locking has been exported, but it's a potential issue. I note that you still use locks with `IsStreaming`

Reply:
Yeah they do, but they are just helpers as well, any inspection on the linked list externally would require a lock. I will remove the lock for the IsStreaming function. It's up to the caller to provide locking protection. And I will export everthing.